src/main/java/no/hvl/dat250/h2020/group5/service/PollService.java

```diff
@@ -71,12 +71,8 @@ public boolean deletePoll(Long pollId, Long userId) {
         .collect(Collectors.toList());
   }
 
-  public List<PollResponse> getAllPolls(Long adminId) {
-    Optional<User> maybeUser = userRepository.findById(adminId);
-    if (maybeUser.isPresent() && maybeUser.get().getIsAdmin()) {
-      return pollRepository.findAll().stream().map(PollResponse::new).collect(Collectors.toList());
-    }
-    return null;
+  public List<PollResponse> getAllPolls() {
+    return pollRepository.findAll().stream().map(PollResponse::new).collect(Collectors.toList());
```

Comment:
Testen her trengs ikke fordi endepunktet som bruker denne er kun tilgjengelig for admins, sant? Bare for å forsikre meg om at jeg har forstått riktig 😅

Reply:
Ja, stemmer 😄. Jeg har brukt `@PreAuthorize` på endepunktet, så det skal være sikret